docs/default.nix

```diff
@@ -88,7 +88,19 @@ rec {
       inherit transformOptions;
       warningsAreErrors = false;
     }).optionsJSON;
+
+  mkSearch =
+    baseHref:
+    nuschtosSearch.packages.mkSearch {
+      optionsJSON = options-json + "/share/doc/nixos/options.json";
+      urlPrefix = "https://github.com/nix-community/nixvim/tree/main";
+      inherit baseHref;
+    };
+in
+{
   man-docs = pkgsDoc.callPackage ./man { inherit options-json; };
+
+  search = mkSearch "/";
 }
```

Comment:
Think we need to inherit `options-json` if that was exported before?

Reply:
good catch